tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -1895,84 +1895,7 @@ def test__get_all_user_datasets_for_index_requires_auth(self):
         response = self.app.get(test_url.url, headers=headers)
         self.assertEqual(response.status_code, 401)
 
-    # ✅
-    def test__get_all_datasets_for_index_with_ontology_expansion_deprecated(self):
```

Comment:
why is this test being removed?

Reply:
see the comment directly below on lns 1900-1902; I confirmed with @MillenniumFalconMechanic that it can be removed now